src/app/contextBroker/orionRestServices.cpp

```diff
@@ -237,9 +237,10 @@ static RestService postServiceV[] =
 {
```

Comment:
CNR entry missing.

With regards to documentaiton:

* modifications in devel manual ("duplicate" the same that is already done for NGSIv1 notifs)
* modifications in NGSIv2 .apib

I'd suggest to defer to a next PR. However, add a comment at #2986 about that missing part.

Reply:
Issue updated